java/benchmarks/src/main/java/glide/benchmarks/clients/jedis/JedisClient.java

```diff
@@ -6,48 +6,60 @@
 import java.util.Set;
 import redis.clients.jedis.DefaultJedisClientConfig;
 import redis.clients.jedis.HostAndPort;
+import redis.clients.jedis.Jedis;
 import redis.clients.jedis.JedisCluster;
 import redis.clients.jedis.JedisPool;
-import redis.clients.jedis.commands.JedisCommands;
+import redis.clients.jedis.util.JedisClusterCRC16;
 
 /** A Jedis client with sync capabilities. See: https://github.com/redis/jedis */
 public class JedisClient implements SyncClient {
-
-    private JedisCommands jedis;
+    boolean isClusterMode;
+    private JedisPool jedisPool;
+    private JedisCluster jedisCluster;
 
     @Override
     public void closeConnection() {
-        // nothing to do
+        jedisPool.close();
+        jedisCluster.close();
     }
 
     @Override
     public String getName() {
         return "Jedis";
     }
 
+    private Jedis getConnection(String key) {
```

Comment:
you are messing up slot key and value key. You call this function with value key, but use it as slot key.

Reply:
I was under the impression that `getConnectionFromSlot` returns the slot from the key value... but you are correct that it just takes the slot key.